db/migrate/20170816143355_add_expense_columns_to_snap_application.rb

```diff
@@ -0,0 +1,29 @@
+class AddExpenseColumnsToSnapApplication < ActiveRecord::Migration[5.1]
+  def change
+    add_column :snap_applications, :rent_expense, :integer
+    add_column :snap_applications, :property_tax_expense, :integer
+    add_column :snap_applications, :insurance_expense, :integer
+    add_column :snap_applications, :utility_heat, :boolean
+    add_column :snap_applications, :utility_cooling, :boolean
+    add_column :snap_applications, :utility_electrity, :boolean
+    add_column :snap_applications, :utility_water_sewer, :boolean
+    add_column :snap_applications, :utility_trash, :boolean
+    add_column :snap_applications, :utility_phone, :boolean
+    add_column :snap_applications, :utility_other, :boolean
+
+    add_column :snap_applications, :dependent_care, :boolean
+    add_column :snap_applications, :medical, :boolean
```

Comment:
hmmm ok, let's potentially revisit this soon, then. I am just noticing as I fill in the PDF that it is hard to sometimes tell which DB fields match to which PDF fields. Hopefully we can get to a place naming-wise where it is more obvious without having to dig through the views and controllers.